test/connection.test.ts

```diff
@@ -488,31 +488,14 @@ describe('Connection', function () {
     })
 
     it('should determine the exchange rate even if it is small', async function () {
-      this.clientPlugin.exchangeRate = 0.000001
+      this.clientPlugin.exchangeRate = 0.0001
       this.clientPlugin.maxAmount = 1000000
       await createConnection({
         ...this.server.generateAddressAndSecret(),
         plugin: this.clientPlugin
       })
     })
 
-    it('should determine the exchange rate even if the maximum packet amount is < 1000', async function () {
-      this.clientPlugin.maxAmount = 99
-      await createConnection({
-        ...this.server.generateAddressAndSecret(),
-        plugin: this.clientPlugin
-      })
-    })
-
-    it('should determine the exchange rate even if it is very very small', async function () {
```

Comment:
Mm I see. Should we add a 10^12 test packet to cover that?

Reply:
I'm on the fence on this one, if we add 10^12 in the test packets why not do 10^15 or 10^18 as well. I think we should keep our current 4 for now 1, 10^3, 10^6, & 10^9 if we need to add 10^12 we can do so later. 

I think not putting in 10^12 is the way to go for now. If we add it then it encourages even greater exchange disparities between connectors.

Reply:
The argument I see for adding 10^12 now is that this PR makes it require 3 digits of precision to work, whereas before it worked with only 1. So without this change, the implementation supports an exchange rate of 10^-9 but this PR would break that.

Reply:
Thats a good point, so would we send 5 test packets to include 10^12 to support this scenario?

Reply:
Yup

Reply:
Added 10^12 support.